src/app/modules/about-us/about-us.component.html

```diff
@@ -43,12 +43,20 @@
       <p class="list-title">THE TEAM</p>
       <div class="people-list-grid">
         <a class="people-profile text-decoration-none" href=" https://in.linkedin.com/in/antaraavasudev" target="_blank">
-          <img class="avatar avatar--128" src="../../../assets/images/antaraa.png" />
+          <picture class="avatar avatar--128">
+            <source srcset="../../../assets/images/antaraa.webp" type="image/webp">
+            <source srcset="../../../assets/images/antaraa.jpg" type="image/jpeg">
+            <img src="../../../assets/images/antaraa.jpg" alt="Antaraa image" />
```

Comment:
@barath46 Put a habit of directly using absolute path wherever you can `/assets/images/....`